### PR TITLE
Update `self.appInfos` in `create/destroyApp()`

### LINKED ISSUE #184: `appInfo` is not updated at all
### Describe the bug
`Qiwis.appInfos` attribute is initialized in the constructor and never updated even when `createApp()` or `destroyApp()` is called.
In consequence, `updateFrames()` call fails for newly created apps.

### Expected behavior
`updateFrames()` should work properly for newly created apps, which are not loaded in the constructor.

### To Reproduce
1. Setup a `Qiwis` system.
2. Call `createApp(name, appinfo)` to add an app.
3. Call `updateFrames(name)`.

### Expected causes and correction methods
Update `appInfos` attribute properly in `createApp()` and `destroyApp()`.

### Screenshots
Screen Shot 2023-06-08 at 19 04 58
Screen Shot 2023-06-08 at 19 04 33

### PR DESCRIPTION
**Pull request by kangz12345**
This resolves #184

### CODE REVIEW

File: qiwis.py [hunk 1]
```diff
@@ -164,7 +164,7 @@ def __init__(
             parent: A parent object.
         """
         super().__init__(parent=parent)
-        self.appInfos = appInfos
+        self.appInfos: Dict[str, AppInfo] = {}
```
Comment (BECATRUE) [inline qiwis.py:198]: We didn't write down any attributes in `Qiwis` class strangely.
I think we need to do this not in this RP, but later.

**Review by BECATRUE — DISMISSED** [review]
LGTM!

File: qiwis.py (hunk 1 above)
Comment (kangz12345) [reply qiwis.py:198]: You mean the attribute docstring? Yeah 😄

Comment (BECATRUE) [reply qiwis.py:198]: Yes👍

**Comment by BECATRUE** [comment]
You can merge this PR, now! @kangz12345

**Comment by kangz12345** [comment]
@BECATRUE Please check this again as I resolved a merge conflict.

**Review by BECATRUE — APPROVED** [review]
LGTM!

**Pull request merged**